### PR TITLE
CSV enhancement

### LINKED ISSUE #43: Update CSV with real information
## Is your feature request related to a problem? Please describe.

We definitely need an appropriate icon and description. 

## Describe the solution you'd like.

We do not have the icon or description at the moment so part of this effort will be to lock them down.
The preflight run may also provide more required updates so watch those results if they are available or we can just create new issues when those results become available.

### PR DESCRIPTION
**Pull request by itroyano**
This PR addresses the following -
1) Adding an icon, description and links to the CSV for user-friendly output to be seen in operatorhub.io/preview
2) passing Preflight bundle check
3) typos and branding.

Fixes #43 

Signed-off-by: Igor Troyanovsky <itroyano@redhat.com>

### CODE REVIEW

**Comment by itroyano** [comment]
Preview:

image

**Comment by acornett21** [comment]
Nit: Remove the `new lines` that github is complaining about in a files. Other than that, I see no issues.

**Comment by itroyano** [comment]
> Nit: Remove the `new lines` that github is complaining about in a files. Other than that, I see no issues.

done. thanks!

File: bundle.Dockerfile [hunk 1]
```diff
@@ -39,5 +41,7 @@ LABEL quay.expires-after=${quay_expiration}
 
 # Copy files to locations specified by labels.
 COPY --from=builder /workspace/bundle/manifests /manifests/
-COPY --from=builder /workspace/bundle/metadata /metadata/
+# Working around an issue where 'make bundle' will remove annotations from /metadata/annotations.yaml, causing a failed certification
+COPY --from=builder /workspace/annotationsWA /metadata/
+#COPY --from=builder /workspace/bundle/metadata /metadata/
```
Comment (samira-barouti) [inline bundle.Dockerfile:46]: can we remove this line?

File: bundle/metadata/annotations.yaml [hunk 2]
```diff
@@ -5,10 +5,14 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: operator-certification-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
```
Comment (samira-barouti) [inline bundle/metadata/annotations.yaml:8]: Do we need this file as well? This is a duplicate of `annotationsWA/annotations.yaml`. Can we keep one of them?

File: bundle.Dockerfile (hunk 1 above)
Comment (itroyano) [reply bundle.Dockerfile]: done.

File: bundle/metadata/annotations.yaml (hunk 2 above)
Comment (itroyano) [reply bundle/metadata/annotations.yaml]: removed it from PR.   I think it's a separate discussion if the file itself needs to be in the codebase.  @bcrochet has a thread on it.

**Review by bcrochet — COMMENTED** [review]
Please explain what 'annotationsWA' means? Specifically, the 'WA' part? We need to have a longer discussion about what these things bring to us, and whether we should be doing these things in the first place.

**Comment by itroyano** [comment]
@bcrochet 

> Please explain what 'annotationsWA' means? Specifically, the 'WA' part? We need to have a longer discussion about what these things bring to us, and whether we should be doing these things in the first place.

Preflight validates some annotations which `make bundle` doesn't generate. specifically `operators.operatorframework.io.bundle.channel.default.v1`

Moreover if I have an `annotations.yaml` which I've modified and added this annotation - running `make bundle` again with actually REMOVE the extra annotations.

As a WA, I'm keeping the annotations file static.  it doesn't have replaceable versions or operator-spec related attributes anyways.

File: bundle.Dockerfile [hunk 3]
```diff
@@ -27,6 +27,8 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=operator-certification-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.bundle.channels.v1: alpha
```
Comment (bcrochet) [inline bundle.Dockerfile:30]: Are we sure this does the right thing? Can labels be specified as 'LABEL key: value'? The others are 'LABEL key=value'.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Check on dockerfile

File: bundle.Dockerfile [hunk 4]
```diff
@@ -39,5 +40,6 @@ LABEL quay.expires-after=${quay_expiration}
 
 # Copy files to locations specified by labels.
 COPY --from=builder /workspace/bundle/manifests /manifests/
-COPY --from=builder /workspace/bundle/metadata /metadata/
+# Working around an issue where 'make bundle' will remove annotations from /metadata/annotations.yaml, causing a failed certification
```
Comment (bcrochet) [inline bundle.Dockerfile:43]: This should no longer be necessary.

**Review by bcrochet — CHANGES_REQUESTED** [review]

File: bundle.Dockerfile (hunk 4 above)
Comment (itroyano) [reply bundle.Dockerfile]: reverted

File: bundle.Dockerfile (hunk 3 above)
Comment (itroyano) [reply bundle.Dockerfile]: reverted

**Comment by itroyano** [comment]
Closing this PR in favor of the up to date consolidated #70

**Pull request closed without merging**